src/game_display.hpp

```diff
@@ -172,6 +172,9 @@ class game_display : public display
 
 	std::set<map_location> units_that_can_reach_goal_;
 
+	std::vector<std::string> reach_map_images_;
```

Comment:
This doesn't seem to be used anymore. Was it meant to be used in `get_reachmap_images` to reduce the number of `image::exists` calls? If it's not necessary anymore, please remove.

Reply:
Also, if this _is_ used by `get_reachmap_images` (and _only_ there), then it should be marked `mutable`.

Reply:
It was meant to use _reach.png_ image variations (e.g. _reach1.png_), as fog does, which was a possibility I tried, but I can see I never needed to use it in my commits, I definitely lost track of what I was doing with the variable. 
Anyways, compiled and tested `get_reachmap_images` using `const` and worked, in 483c62402.